### PR TITLE
Add arrow draw

### LINKED ISSUE #607: Spining arrow
New toss that spins an arrow.
Can be used to choose a person within a group and we can add also numbers so it counts as a casino roulette

### PR DESCRIPTION
**Pull request by Mariocj89**
A new draw that spins an arrow
Closes #607 

We need:
1) Better name for the draw
2) better image for the draw
3) Translations

### CODE REVIEW

**Comment by palvarez89** [comment]
What about:
- Spinner Draw (from the [Twister rules](https://www.math.uni-bielefeld.de/~sillke/Twister/rules/): "The referee spins the spinner...")
- Spinner Arrow
- Lucky Spinner
- Lucky Arrow

**Comment by Mariocj89** [comment]
:+1: Spinner Draw

**Comment by Mariocj89** [comment]
Any suggestion for the translation tobspanish?

**Comment by Mariocj89** [comment]
Please review :)

File: locale/es/LC_MESSAGES/django.po [hunk 1]
```diff
@@ -254,6 +262,31 @@ msgstr "Tiarada sin título"
...
+"            You reading this text does not fix the problem.\n"
+"            In fact, you're likely pissed off at the moment.\n"
+"            But take solate, <strong>top men</strong> are now aware.\n"
+"        "
+msgstr ""
+"\n"
+"            Ups, algo se ha roto en nuestro servidor.\n"
+"            El que leas esto no va a solucionar el problema.\n"
+"            De hecho, debes estar bastante enfadado ahora mismo.\n"
+"            Tomalo con calma, <strong>hombretón</strong> hemos sido "
```
Comment (dnaranjo89) [inline locale/es/LC_MESSAGES/django.po]: lol

File: locale/es/LC_MESSAGES/django.po [hunk 2]
```diff
@@ -964,11 +998,15 @@ msgstr "Asocia elementos de dos listas"
 msgid "Cleaning the house? Allocate the tasks within your housemates"
 msgstr "¿Toca limpiar la casa? Reparte las tareas con tus compañeros"
 
-#: web/templates/index.html:81
+#: web/templates/index.html:72
+msgid "Just hit the spinner and you say the meaning!"
+msgstr ""
```
Comment (dnaranjo89) [inline locale/es/LC_MESSAGES/django.po]: missing translation

File: web/templates/draws/display_draw.html [hunk 3]
```diff
@@ -162,8 +164,8 @@
     <form class="clearfix" method="post" id="draw-form">
         <!-- Draw Heading -->
         <div class="clearfix draw-heading">
-            <a href="{% url 'index' %}" class="back-arrow col-xs-2 col-sm-2 text-center">
-                <img src="{% static "img/back_arrow.png" %}" alt="{% trans "back" %}"/>
+            <a href="{% url 'index' %}" class="back-spinner col-xs-2 col-sm-2 text-center">
+                <img src="{% static "img/back_spinner.png" %}" alt="{% trans "back" %}"/>
```
Comment (dnaranjo89) [inline web/templates/draws/display_draw.html]: Wrong replace

**Comment by Mariocj89** [comment]
@dnaranjo89 Ready

**Pull request merged**